community/civilized-discussion.md

```diff
@@ -0,0 +1,112 @@
+
+* Name-calling
+* Ad hominem attacks
+* Responding to a post’s tone instead of its actual content
+* Knee-jerk contradiction
+
+Instead, provide thoughtful insights that improve the conversation.
+
+Write your opinions **as opinions**. Rather than saying "everyone knows Python is the best language for beginners" or "Python is the best language for beginners", say "I feel that Python is the best language for beginners because ...". We take this seriously as it's an innocent looking habit that can be extremely disruptive to an online community. If you constantly write opinions as facts, we will ask you to stop and refer you to these guidelines. If you continue, we may choose to ban you.
+
+<a name="derailing"></a>
+
+## [Stay on topic - don’t derail](#derail)
+
+When posting, always think about whether you’re moving the conversation in the direction that the original poster intended, or whether you’re pulling it in a different direction. If it’s the latter, it's generally better to create a new topic.
+
+For example, if there’s a topic on cats, but you prefer dogs, don’t create a post saying “cats are rubbish - I like dogs” - instead create a new topic on dogs with your favorite puppy pictures in.
+
+<a name="participate"></a>
+
+## [Your Participation Counts](#participate)
+
+The conversations we have here set the tone for every new arrival. Help us influence the future of this community by choosing to engage in discussions that make this forum an interesting place to be &mdash; and avoiding those that do not.
+
+Discourse provides tools that enable the community to collectively identify the best (and worst) contributions: bookmarks, likes, flags, replies, edits, watching, muting and so forth. Use these tools to improve your own experience, and everyone else’s, too.
+
+Let’s leave our community better than we found it.
+
+<a name="flag-problems"></a>
+
+## [If You See a Problem, Flag It](#flag-problems)
+
+Moderators have special authority; they are responsible for this forum. But so are you. With your help, moderators can be community facilitators, not just janitors or police.
+
+When you see bad behavior, don’t reply. Replying encourages bad behavior by acknowledging it, consumes your energy, and wastes everyone’s time. _Just flag it_. If enough flags accrue, action will be taken, either automatically or by moderator intervention.
+
+In order to maintain our community, moderators reserve the right to remove any content and any user account for any reason at any time. Moderators do not preview new posts; the moderators and site operators take no responsibility for any content posted by the community.
+
+<a name="be-civil"></a>
+
+## [Always Be Civil](#be-civil)
+
+Nothing sabotages a healthy conversation like rudeness:
+
+* Be civil. Don’t post anything that a reasonable person would consider offensive, abusive, or hate speech.
+* Keep it clean. Don’t post anything obscene or sexually explicit.
```

Comment:
To me using "clean" here feels very judgemental. While sexually explicit content has no place in this forum, it's not "dirty", which this post implies. You can remove the "Keep it clean" and it works just as well without that judgement.

Reply:
It's also not rude to be sexually explicit, but it _is_ rude to be _out of place_. Exercism is not a place to be sexually explicit or obscene.